move/walrus_site/sources/site.move

```diff
@@ -212,4 +243,44 @@ module walrus_site::site {
         } = site;
         id.delete();
     }
+
+    #[allow(lint(self_transfer))]
+    /// Define a Display for the Site objects.
+    public fun set_site_display(publisher: &Publisher, ctx: &mut TxContext) {
+        // Check if the `Publisher` has the authority over the Display.
+        // Checks if the type is from the same module, hence the
+        assert!(publisher.from_module<Site>(), ENotAuthorized);
+        // Checks if the type is from the same package.
+        assert!(publisher.from_package<Site>(), ENotAuthorized);
```

Comment:
if `from_module` is true, `from_package` is automatically true, check is redundant.